src/gs-details-page.c

```diff
@@ -446,31 +446,28 @@ gs_details_page_refresh_progress (GsDetailsPage *self)
 		gtk_widget_set_visible (self->button_cancel, FALSE);
 		break;
 	}
-	if (app_has_pending_action (self->app)) {
+	if (plugin_or_app_has_pending_action (self, self->app)) {
```

Comment:
Commit titles in g-s (usually) only start by the module when it's a change in the plugins. Also seems like this commit should rather be squashed into the others you did before?

Reply:
Fixed locally.

This conditional only needs to change when we support copying so I don't think it's necessarily better to move the support function to the previous commit given it isn't used yet by then (and would trigger an "unused function" warning when building at that point).